api/all/src/jmh/java/io/opentelemetry/api/baggage/propagation/JdkStringOperationsBasedPropagator.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.baggage.propagation;
+
+import static java.util.Collections.singletonList;
+
+import io.opentelemetry.api.baggage.Baggage;
+import io.opentelemetry.api.baggage.BaggageBuilder;
+import io.opentelemetry.api.baggage.BaggageEntryMetadata;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.propagation.TextMapGetter;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import io.opentelemetry.context.propagation.TextMapSetter;
+import java.util.Collection;
+import java.util.List;
+import javax.annotation.Nullable;
+
+/** JDK string-ops based W3C baggage propagator. For comparison purposes. */
+class JdkStringOperationsBasedPropagator implements TextMapPropagator {
```

Comment:
This should be deleted before this PR will be ready to merge. There's no reason to keep it around if we're going with the new approach.

Reply:
I was thinking about leaving it just as a "reference" implementation that could be compared against in case of a newer implementation. Yet if you say it should be deleted, it will be :)

Reply:
yeah, we don't want to leave dead code around. :)